keel-core/src/main/kotlin/com/netflix/spinnaker/keel/lifecycle/LifecycleEvent.kt

```diff
@@ -21,6 +21,8 @@ import java.time.Instant
  *   the user.
  * [timestamp] is nullable so that the repository can insert the timestamp when
  *   the event is stored.
+ * [monitor] true if this event is the first in the chain and needs to be monitored.
```

Comment:
There's _a lot_ of detail encoded in this name. How about `isMonitoringTrigger` instead, for a bit more context?

Reply:
how about: `startMonitoring: Boolean`?

Reply:
Sure!